foodalert/static/foodalert/js/pages/student/notification-page.vue

```diff
@@ -8,25 +8,36 @@
         Select how you will like to receive notifications.
         Please choose at least one.
       </p>
-      <text-notif accord_id="text" type="text"
-        label="Enter a phone number" :subid="subid"
-        :serverData="{ text: notif_info.sms_number,
-          verified: notif_info.number_verified,
-          send_sms: notif_info.send_sms}"
-        :requestUpdate="requestUpdate"
-        errorDesc="Carrier rates may apply">
-        <template #disclaimer>
-          Only US numbers are supported at this time. Carrier rates may apply.
-        </template>
-        <template #update-note>
-          Remove or update your phone number.
-        </template>
-        <template #unverifNotifText>
-          We sent a verification text to {{notif_info.sms_number}}.
-          Please reply YES to finish signup.
-          <br />
-        </template>
-      </text-notif>
+      <div v-bind:class="{'overlay': (notif_info.twilio_stop)}">
+        <text-notif accord_id="text" type="text"
+          label="Enter a phone number" :subid="subid"
+          :serverData="{ text: notif_info.sms_number,
+            verified: notif_info.number_verified,
+            send_sms: notif_info.send_sms}"
+          :requestUpdate="requestUpdate"
+          errorDesc="Carrier rates may apply">
+          <template #disclaimer>
+            Only US numbers are supported at this time. Carrier rates may apply.
+          </template>
+          <template #update-note>
+            Remove or update your phone number.
+          </template>
+          <template #unverifNotifText>
+            We sent a verification text to {{notif_info.sms_number}}.
+            Please reply YES to finish signup.
+            <br />
+          </template>
+          <template #twilio-warning>
+            <b-form-text text-variant="primary"
+              style="font-size: 14px; font-weight: 800;">
+              Warning: Please respond back to our messages
+              <span v-if="twilioNum!='NONE'"> at {{twilioNumber}}</span>
```

Comment:
Sounds like the variable name is now 'twilioNumber' so this will need to be changed.